content/pt/docs/concepts/configuration/pod-overhead.md

```diff
@@ -0,0 +1,197 @@
+---
+reviewers:
+- dchen1107
+- egernst
+- tallclair
+title: Pod Overhead
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+Quando executa um Pod num nó, o próprio Pod usa uma quantidade de recursos do sistema. Estes
+recursos são adicionais aos recursos necessários para executar o(s) _container(s)_ dentro do Pod.
+Sobrecarga de Pod, do inglês _Pod Overhead_, é uma funcionalidade que serve para contabilizar os recursos consumidos pela
+infraestrutura do Pod para além das solicitações e limites do _container_.
+
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+No Kubernetes, a sobrecarga de _Pods_ é definido no tempo de
+[admissão](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
+de acordo com a sobrecarga associada à
+[RuntimeClass](/docs/concepts/containers/runtime-class/) do _Pod_.
+
+Quando é ativada a Sobrecarga de Pod, a sobrecarga é considerada adicionalmente à soma dos 
+pedidos de recursos do _container_ aquando do agendamento de um Pod. Semelhantemente, o _kubelet_
+vai incluir a sobrecarga do Pod aquando do dimensionamento do cgroup do Pod, e aquando de
+levar a cabo a classificação de despejo do Pod.
```

Comment:
I think it would look better as follows:
```suggestion
Quando é ativada a Sobrecarga de Pod, a sobrecarga é considerada adicionalmente à soma das
solicitações de recursos do _container_ ao agendar um Pod. Semelhantemente, o _kubelet_
incluirá a sobrecarga do Pod ao dimensionar o cgroup do Pod e ao
executar a classificação de despejo do Pod.
```